src/index.css

```
@@ -231,6 +231,7 @@ a:hover {
 #react-burger-menu-btn {
   width: inherit !important;
   left: unset !important;
+  height:inherit !important;
```

- Add spacing (please install Prettier to avoid this in the future) 
- Move one line above, so it'd be right below the element's `width`

```suggestion
  height: inherit !important;
```